relay_handler.js

```diff
@@ -372,6 +392,78 @@ function handleFrameLazily(frame) {
             relayDirection: 'in'
         }));
     }
+
+    if (frame.type === v2.Types.CallRequest) {
+        self._observeCallReqFrame(frame, now);
+    } else if (frame.type === v2.Types.CallRequestCont) {
+        self._observeCallReqContFrame(frame, now);
+    // } else { TODO: log
+    }
+};
+
+LazyRelayInReq.prototype._observeCallReqFrame =
+function _observeCallReqFrame(frame, now) {
+    var self = this;
+
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.inbound.request.size',
+        'counter',
+        frame.size,
+        new stat.InboundRequestSizeTags(
+            self.callerName,
+            self.serviceName,
+            self.endpoint
+        )
+    ));
+
+    self.channel.emitFastStat(self.channel.buildStat(
```

Comment:
TODO: No garantuee that we send the frame out succesfully. frame write could fail.